internal/querynode/watch_dm_channels_task.go

```diff
@@ -244,11 +244,20 @@ func (w *watchDmChannelsTask) LoadGrowingSegments(ctx context.Context, collectio
 		zap.Int64("collectionID", collectionID),
 		zap.Int64s("unFlushedSegmentIDs", unFlushedSegmentIDs),
 	)
-	_, err := w.node.loader.LoadSegment(w.ctx, req, segmentTypeGrowing)
+	loaded, err := w.node.loader.LoadSegment(w.ctx, req, segmentTypeGrowing)
 	if err != nil {
 		log.Warn("failed to load segment", zap.Int64("collection", collectionID), zap.Error(err))
 		return nil, err
 	}
+	for _, segmentID := range loaded {
+		segment, err := w.node.metaReplica.getSegmentByID(segmentID, segmentTypeGrowing)
+		if err != nil {
+			return nil, err
```

Comment:
log warning and just continue here, it may have been replaced with a sealed segment